src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonOffenderEventListener.kt

```diff
@@ -16,7 +16,7 @@ class PrisonOffenderEventListener(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  @JmsListener(destination = "incentives", containerFactory = "hmppsQueueContainerFactoryProxy")
+  @SqsListener("incentives", factory = "hmppsQueueContainerFactoryProxy")
```

Comment:
I spotted https://github.com/ministryofjustice/hmpps-prisoner-to-nomis-update/pull/164 - which we could also include if folks think it would be helpful?

Reply:
Good shout - raised https://dsdmoj.atlassian.net/browse/INC-1198